index.js

```diff
@@ -1,12 +1,13 @@
+const tty = require('tty');
```

Comment:
Colorette is an ES module, so this should be changed to use `import`.

Reply:
will that be compatible with current dual publishing of CJS and ES versions? do you autoreplace?

Reply:
Nope, looks like the time to upgrade that script has finally come! I can take care of that after we merge this.

Reply:
done